app/assets/javascripts/oxalis/view/tracing_view.js

```diff
@@ -30,37 +18,27 @@ const registerWebGlCrashHandler = canvas => {
   );
 };
 
-class TracingView extends React.PureComponent<Props> {
-  handleContextMenu(event: SyntheticInputEvent<>) {
+export default function TracingView() {
+  const handleContextMenu = (event: SyntheticInputEvent<>) => {
     // hide contextmenu, while rightclicking a canvas
     event.preventDefault();
-  }
-
-  render() {
-    const divClassName = classnames({
-      "zoomstep-warning": this.props.isVolumeTracingDisallowed,
```

Comment:
This had no effect anyway or why did you remove it?

Reply:
> I also noticed that the zoomstep-warning for volume tracings no longer worked due to golden layout (red border), which is why I removed it. We're showing an error toast anyways.

Yes, it even screwed up the layout a little bit :/